sceptre/cli/dump.py

```diff
@@ -114,6 +114,47 @@ def dump_template(ctx, to_file, no_placeholders, path):
             write(template, output_format, no_colour=True)
 
 
+@dump_group.command(name="vars")
+@click.argument("path")
+@click.option(
+    "--to-file", is_flag=True, help="If True, also dump the template to a local file."
```

Comment:
It's not a template. 
```suggestion
    "--to-file", is_flag=True, help="If True, also dump the vars to a local file."
```